src/ExternalTrackerGoogleAnalyticsBundle/Bridge/Chameleon/ExternalTracker/ExternalTrackerGoogleAnalyticsGa4.php

```diff
@@ -0,0 +1,187 @@
+<?php declare(strict_types=1);
+
```

Comment:
This should have the license header.

Reply:
:heavy_check_mark: Has a license header now.